uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -7,6 +7,7 @@ define([
   'common/moment',
   'common/services/api/option-group',
   'common/services/hr-settings',
+  'common/services/session',
```

Comment:
This file had lot of violations of the angular style guide. As we touched it, should be fixed.